src/main/resources/tags/errors/mismatch.tag

```diff
@@ -1,8 +1,11 @@
 type: text
+title: :x: Floodgate Key Mismatch
 issues: Please connect through the official Geyser
+color: failure
+button: [WinSCP](https://winscp.net/eng/index.php)
 
 ---
 
 This error is caused by a Floodgate key.pem mismatch, most likely between your proxy and your backend servers.
 Make sure that the key.pem file in every Floodgate config folder is the same file.
-If this still doesn't work and you're using an FTP client, try using the [WinSCP](https://winscp.net/eng/index.php) client.
\ No newline at end of file
+If this still doesn't work and you're using an FTP client, download the WinSCP client using the link below and use that instead.
```

Comment:
I don't think this one should be a button, we're not encouraging a click like we are others.